pkg/apis/componentconfig/types.go

```diff
@@ -288,6 +288,8 @@ type KubeletConfiguration struct {
 	NodeLabels map[string]string `json:"nodeLabels"`
 	// nonMasqueradeCIDR configures masquerading: traffic to IPs outside this range will use IP masquerade.
 	NonMasqueradeCIDR string `json:"nonMasqueradeCIDR"`
+	// enable gathering custom metrics.
```

Comment:
this comment should start with enableCustomMetrics (json field name) because it will be used to generate documentations